forem/forem
# Remove "actions" from Member Index View that have not been implemented as yet

## What type of PR is this? (check all applicable)

- [x] Wrap Up Task

## Description
In an effort to wrap up the Member Index View, we will be removing the Actions that had been stubbed behind the feature flag with the intention of implementing it. 

## Related Tickets & Documents
-

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings
You should no longer see the options that were removed (Send an email, Write a note, Unpublish all posts, Banish Users) within the dropdown.

<img width="478" alt="Screenshot 2022-07-13 at 15 27 45" src="https://user-images.githubusercontent.com/2786819/178744914-68b2a510-fffd-40c3-99bc-d74eb452ffe4.png">

### UI accessibility concerns?

None.

## Added/updated tests?

- [x] No, and this is why: this was stubbed UI elements that resided behind a feature flag and was not hooked up as yet.

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
